ansible/roles/software/ckan/catalog/ckan-app/tasks/worker.yml

```diff
@@ -28,6 +28,18 @@
     - usr/bin/celery-mem-check.sh
   when: catalog_ckan_worker_type == 'qa_worker'
 
+- name: schedule ckan-clean-harvest-logs cron
+  cron:
+    name: "clean harvest logs in db"
+    cron_file: ckan
+    job: supervisorctl start ckan-clean-harvest-logs > /dev/null
+    minute: "30"
+    hour: "2"
+    user: root
+    disabled: "{{ not crons_enabled }}"
+    state: {{ catalog_enable_clean_harvest_logs | ternary('present', 'absent') }}
```

Comment:
Can we do:
```
disabled: {{ not crons_enabled and not catalog_enable_clean_harvest_logs }}
state: present
```

I feel this way logic is easier.

Reply:
I [just changed](https://github.com/GSA/datagov-deploy/pull/2327/commits/7faf22e9b3e4f04d1fce6fe939625bd17776a5e3) @FuhuXia. I use `or` instead `and` because I think that's is the right way here.

Reply:
👍  
Sorry for the and/or mistake.